person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  attr_reader(:id)
+  attr_accessor(:name, :age)
```

Comment:
- [ ] attr_accessor and attr_reader are used without parentheses this will make it look like attr_accessor :name, :age

Reply:
Fixed